openmc/deplete/abc.py

```diff
@@ -265,8 +265,8 @@ def get_material_rates(self, mat_id, nuc_index, react_index):
             Ordering of reactions
         """
 
-    def divide_by_adens(self, number):
-        """Normalize reaction rates by number of nuclides
+    def divide_by_atoms(self, number):
+        """Normalize reaction rates by number of atoms
```

Comment:
Subtle but important change. `nuclide` is not the correct unit to use here because there are a fixed number of nuclides that exist (e.g. U235, Xe135, etc.) ; atom is the unit we use to describe how much of a nuclide we have (2 atoms of U235, 500 atoms of Xe135)